src/main/java/com/amazon/opendistroforelasticsearch/ad/rest/RestDeleteAnomalyDetectorAction.java

```diff
@@ -78,12 +78,19 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
 
     @Override
     public List<Route> routes() {
+        return ImmutableList.of();
```

Comment:
Can we sink this into a base class and avoid the copy-paste?

Reply:
Thats a great idea. I can add this in OpenSearch in the base class. 
I'll do a quick follow up after this PR.

Reply:
Created an issue to take care of this: https://github.com/opensearch-project/OpenSearch/issues/694